### ISSUE TITLE
ChrysaLisp on/in SBTCVM?

### ISSUE THREAD

**Issue by no-identd** [vygr/ChrysaLisp#29]
I just found @SBTCVM and this sounds like the kind of thing one might consider a fun idea. :)

**Comment by neauoire** [comment]
This is great, thanks for sharing!

**Comment by no-identd** [comment]
Glad to do so! A more esoteric albeit somewhat related idea would consist of targeting the infamous DSSP language, see here for details on that:

http://brokestream.com/daf.txt

**Comment by neauoire** [comment]
That reminds me of the Nock* programming language.

**Comment by vygr** [comment]
This did appeal to my Forth masochistic streak ! ;)

I'm not sure I think the 'one word, one action' idea is worth it. Especially when it comes to the need to write more words for IF THEN ELSE cases.

Maybe it's just a Russian thing ;) Apologies to any Russians reading !

**Comment by vygr** [comment]
Forth is a truly great idea. Everyone should learn it, just for the way it turns your brain into a pretzel.

There is something extremely powerful in the dual stack architecture and hardware implementations (I did an FPGA design for a Forth core based on the J1) can be very elegant. G144 anyone...

But having looked at Forth and doing a compiler for it, I eventually retreated to the safety of something more conventionally structured.

Without Forth I would never had tried Lisp ! They are like the Ying and Yang of languages, more I think than any other languages. Everything else occupies the center ground, Forth and Lisp define the outer limits.

And let not even get started on <APL>.... ;) (insert your favorite flame bate language here...)
